cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -453,6 +455,19 @@ func isNodeClientCertWithAttestation(csr *capi.CertificateSigningRequest, x509cr
 	return true
 }
 
+func getInstanceMetadata(inst *compute.Instance, key string) string {
+	if !(inst != nil && inst.Metadata != nil && inst.Metadata.Items != nil) {
```

Comment:
it would be more natural for this to read as

if inst == nil || inst.Metadata == nil || inst.Metadata.Items == nil ?